tools/scripts/refresh-ubuntu-amis.py

```diff
@@ -22,13 +22,13 @@ def get_ubuntu_ami(table: List[List[str]], release: str, region: str) -> Union[N
     def filters(line: List[str]) -> bool:
         return all(
             [
-                line[0] == region,
-                line[1] == release,
-                line[3] == "amd64",
+                line[6] == region,
+                line[0] == release,
+                line[5] == "amd64",
                 # Only use EBS, not instance-store.
-                line[4] == "hvm:ebs-ssd",
+                line[4] == "ebs-ssd",
                 # Only use HVM virtualization, not paravirtualization.
-                line[7] == "hvm",
+                line[10] == "hvm",
```

Comment:
I suggest reordering sorting these by index, rather than preserving the old sort but with new indices